api/api.md

```diff
@@ -146,4 +146,6 @@ following these industry best practices.
 
 <!-- include(endpoints/space.md) -->
 
-<!-- include(endpoints/centers.md) -->
\ No newline at end of file
+<!-- include(endpoints/centers.md) -->
+
```

Comment:
remove commented code..

Reply:
I have added neighborhoods section which is not present in the document previously. To add that file we have to add above line in api.md file which is not a commented code.